canonical-websites/design.ubuntu.com
# Upgrade to Vanilla framework v1.6.3

## Done
Upgrade to Vanilla framework v1.6.3 and removed un-needed styling.

## QA
- Run `./run`
- Go to http://localhost:8011
- Check the site over for styling issues

## Issue / Card
Fixes https://github.com/vanilla-framework/vanilla-framework/issues/1465
